protocol/protocol.proto

```diff
@@ -659,7 +659,10 @@ message Minipool {
 }
 
 message Tags {
-    map<string, string> tags = 1;
+    MessageInteractionType message_interaction_type = 1;
+    GroupMentionType group_mention_type = 2;
+    repeated bytes mentioned_user_addresses = 3;
+    repeated bytes participating_user_ids = 4;
```

Comment:
What is the difference between an `address` and `user_id`?

Reply:
(rest of the file uses _address to refer to the user)

Reply:
shoot - yeah i'll fix that.